### PR TITLE
consume sha in property file in opendjdk build.xml

### PR DESCRIPTION
**Pull request by xius666**
related to issue #3108 
contains work from #3083 
test is here https://github.com/adoptium/aqa-tests/pull/3107#issuecomment-954281530

### CODE REVIEW

**Comment by llxia** [comment]
https://github.com/adoptium/aqa-tests/pull/3083 is merged. Please rebase the change. Thanks

**Comment by xius666** [comment]
@llxia fix it pl review

**Comment by llxia** [comment]
Thanks @xius666 . Can you do two tests:
- without using testenv.properties
- using testenv.properties with JDK_REPO and sha

**Comment by xius666** [comment]
https://ci.adoptopenjdk.net/job/Grinder/2408/ 👍 

https://ci.adoptopenjdk.net/job/Grinder/2410/ 👍

File: testenv/testenv.properties [hunk 1]
```diff
@@ -10,8 +10,8 @@ ADOPTOPENJDK_SYSTEMTEST_REPO=https://github.com/adoptium/aqa-systemtest.git
 ADOPTOPENJDK_SYSTEMTEST_BRANCH=master
 JDK8_REPO=https://github.com/adoptium/jdk8u.git
 JDK8_BRANCH=master
-JDK11_REPO=https://github.com/adoptium/jdk11u.git
-JDK11_BRANCH=master
+JDK11_REPO=https://github.com/xius666/jdk11u
+JDK11_BRANCH=71f2c75111e87091616f0f3b86bea6c4d345dad1
```
Comment (llxia) [inline testenv/testenv.properties:14]: Could you rever your testing change? Thanks

**Review by smlambert — APPROVED** [review]
thanks @xius666 !

**Review by llxia — APPROVED** [review]
Thanks @xius666

**Pull request merged**